src/Components/ArticleDetailPage.jsx

```diff
@@ -0,0 +1,44 @@
+import React, {useState, useEffect} from 'react'
+import {useParams} from 'react-router-dom'
+import axios from 'axios'
+import './ArticleDetailPage.css'
+
+function ArticleDetailPage() {
+    const { article_id} = useParams()
+    const [article, setArticle] = useState(null)
+    const [loading, setLoading] = useState(true)
+    const [error, setError] = useState(null)
+
+    useEffect(() => {
+        const fetchArticle = async () => {
+           try {
+            const response = await axios.get(`https://sams-news.onrender.com/api/articles/${article_id}`)
```

Comment:
I would extract this logic out into a util function and store this in a file that house all of your api requests in one place - get this function to return the article and then you can set it here in the state